modules/gdnative/godot/godot_quat.h

```diff
@@ -35,25 +35,56 @@ extern "C" {
 #endif
 
 #include <stdint.h>
+#include <wchar.h>
```

Comment:
instead of `#include <wchar.h>` you should `#include "godot_string.h"` where string is used.

This file doesn't care if string is based on wchars, chars or bools 😄